scps/SCP-1.md

```diff
@@ -0,0 +1,14 @@
+## SCP-1: SCP Categories and Nomenclature
+
+SCPs can pertain to several different categories:
+
+1. *Architecture*: relates to the internal architecture and software engineering of OrbitDB. Proposals
+regarding performance, resiliency, and feature sets should be submitted in this category.
+2. *Developer Experience*: relates to the general UX a contributor has when _developing OrbitDB_ itself. Proposals
+regarding CI/CD, tests, linting, and *internal* documentation should be filed here
+3. *Community*: relates to the community. Proposals pertaining to external documentation, code of conduct, community
+engagement, and the SCP process should be submitted in this category.
+
+### Numbering
+
+SCPs should be numbered the same as the pull request which they are introduced.
```

Comment:
What about the numbering offset that would happen due to issues and/or minor PRs/fixes. We would be skipping quite a lot of numbers over time.